node/node-requesttracker-rest.py

```diff
@@ -56,7 +56,7 @@ def parse_ticket_lines(lines):
             tickets = parse_ticket_lines(tickets[2:])
 
             for id, subject in tickets.iteritems():
-                details = urlopen(Request(uri + "/REST/1.0/ticket/" + id + "/show")).readlines()[2:-1]
+                details = urlopen(Request(uri + "/REST/1.0/ticket/" + id + "/show", headers={"Referer":uri})).readlines()[2:-1]
```

Comment:
```suggestion
                details = urlopen(Request(uri + "/REST/1.0/ticket/" + id + "/show", headers={"Referer": uri})).readlines()[2:-1]
```

Is it more readable? 🤷 

But at least it's PEP8 compliant 😋